packages/component-library/src/Header/index.module.scss

```diff
@@ -219,4 +219,87 @@
       }
     }
   }
+
+  .communityLinkList {
+    display: flex;
+    flex-flow: column nowrap;
+    gap: theme.spacing(4);
+
+    .title {
+      @include theme.text("lg", "medium");
+
+      color: theme.color("heading");
+    }
+
+    .communityGrid {
+      display: grid;
+      grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+      gap: theme.spacing(3);
+      width: 100%;
+
+      @include theme.breakpoint("sm") {
+        grid-template-columns: repeat(auto-fit, minmax(160px, 1fr));
+        gap: theme.spacing(2);
+      }
+    }
+
+    .communityCard {
+      aspect-ratio: 1;
+      min-height: 160px;
+      padding: theme.spacing(6);
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      text-align: center;
+      transition:
+        transform 0.2s ease,
+        box-shadow 0.2s ease;
+
+      &:hover {
+        transform: translateY(-2px);
+        box-shadow: 0 8px 25px rgb(0 0 0 / 15%);
+      }
```

Comment:
This hover styling appears nowhere else in the design system; it's inconsistent and we should make this match everything else